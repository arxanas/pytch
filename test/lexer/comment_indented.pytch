let foo =
  # Comment with an indent.
  bar
