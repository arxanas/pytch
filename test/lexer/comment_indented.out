'let'
leading whitespace ' '
identifier 'foo'
leading whitespace ' '
'='
trailing newline '\n'
leading whitespace '  '
leading comment '# Comment with an indent.\n'
leading whitespace '  '
identifier 'bar'
trailing newline '\n'
the end of a 'let' binding
the end of the file
